contributors/design-proposals/service-policy.md

```diff
@@ -0,0 +1,68 @@
+# Service Policy Proposal
+
+## Abstract
+
+ServicePolicy allows cluster administrators to control the creation and validation of a service.
+The intent of ServicePolicy is to manage the services.
+
+## Motivation
+
+As we know, field level control over allowed fields in services does not exist yet. we can not limit
+the ip range for different users. and if we don't want people use the node port directly, we can not
+implement it although the podsecuitypolicy can limit the "hostPort".
+
+## Goals
+
+1.  Associate [service accounts](../design-proposals/service_accounts.md), groups, and users with
```

Comment:
How do you imagine tying a service account to one of these ServicePolicy objects? I don't currently see that in this proposal.

Presumably there's another "binding" type resource that's missing?

Reply:
> How do you imagine tying a service account to one of these ServicePolicy objects? I don't currently see that in this proposal.
Presumably there's another "binding" type resource that's missing?

Reference to PodSecurityPolicy, ServicePolicy may be need "binding" type resource too.